buildspec.saas.yml

```diff
@@ -22,6 +32,17 @@ phases:
     commands:
       - docker push rudderlabs/rudder-transformer:$VERSION
       - docker push rudderlabs/rudder-transformer:latest
+  change_version:
+      - git clone https://$GITHUB_TOKEN@github.com/rudderlabs/rudder-devops.git
```

Comment:
In the future, we will enable 2FA on all github accounts. Can you confirm if clone and push works with 2FA ?

Reply:
Yes, cloning with PAT works even if 2FA is enabled.